src/gss_ntlmssp.h

```diff
@@ -180,6 +180,10 @@ void gssntlm_int_release_cred(struct gssntlm_cred *cred);
 size_t gssntlm_get_attrs_count(const struct gssntlm_name_attribute *attrs);
 int gssntlm_copy_attrs(const struct gssntlm_name_attribute *src,
                        struct gssntlm_name_attribute **dst);
+int gssntlm_append_attr(const char *attr_name, gss_buffer_t attr_value,
```

Comment:
These functions are currently used only for tests, so let's leave them as static in ntlmssptest.c

Reply:
ok, but must probably you will want to move them back if more attributes will be used by library.
currently we use only one "urn:gssntlmssp:sids" so 'append' operation is not needed.